tomosterlund/qalendar
# Fix/close flyout on click outside

**To test**

1. The event flyout should always close on click, except if:
a) The click occurs on a child of the flyout itself
b) The click occurs on another calendar event, or one of its children.

2. The above should be true for day, week and month modes.
